src/flow/ui/screens/LogoutScreen.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button, Text } from 'react-native';
+
+import ScreenView from '../../../components/ScreenView';
+import InitialRouteNameState from '../../../navigation/initialRouteNameState/InitialRouteNameState';
+
+import useFlowStateContext from '../../sm/useFlowStateContext';
+import LandingScreenFragment from './LandingScreenFragment';
+
+const LogoutScreen = () => {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const [_step1Value, _setStep1Value, resetStep1Value] = useFlowStateContext('step1');
+  const [_step2Value, _setStep2Value, resetStep2Value] = useFlowStateContext('step2');
+  const [_step3Value, _setStep3Value, resetStep3Value] = useFlowStateContext('step3');
+  /* eslint-enable @typescript-eslint/no-unused-vars */
+  const handleConfirmLogoutButtonPress = () => {
+    resetStep1Value();
+    resetStep2Value();
+    resetStep3Value();
+    // Show the Android landing screen fragment to emulate navigation behavior.
+    LandingScreenFragment.showAsync();
```

Comment:
Note for ourselves:

We probably want to make sure the LandingScreenFragment show/hide is only done solely on one side (i.e fully in native/fully in RN), or else maintainability will be a problem as logic is separated across